src/components/Header/Header.scss

```diff
@@ -1,52 +1,102 @@
-@import "../../utils/mixins";
-@import "../../utils/normalize";
+@import "../../App.scss";
 
 .header {
   box-sizing: border-box;
   display: flex;
   justify-content: space-between;
   align-items: center;
-  background-color: #fff;
-  box-shadow: 0px 1px 0px #E2E6E9;
 
-  &-container {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-  }
+  font-style: normal;
+  font-weight: 700;
+  font-size: 12px;
 
-  &-logo {
-    display: flex;
-    margin: 12px;
-  }
+  letter-spacing: 0.04em;
+
+  box-shadow: 0px 1px 0px #E2E6E9;
 
-  &-icon {
-    width: 48px;
-    height: 48px;
+  &-logo {
+    display: flex; 
+    margin-left: 16px;
 
     @include onDesktop {
-      width: 64px;
-      height: 64px;
+      margin-left: 24px;
     }
   }
 }
 
 .nav {
+  width: 100%;
+  display: flex;
+  align-items: center;
 
   &-list {
     display: flex;
     justify-content: center;
     list-style: none;
+    margin: 0;
   }
 
   &-item {
     display: flex;
     margin-right: 64px;
+
+    @include onTablet {
+      margin-right: 32px;
+    }
+
+    &:last-child {
+      margin: 0;
+    }
```

Comment:
It's better to use a gap to separate these items, it's already supported by all browsers
<img width="780" alt="image" src="https://github.com/fe-feb23-team3/fe-product-catalog/assets/121519458/fb0cd7cb-5514-4dd8-b8a5-ed27a905caca">